### PR TITLE
Updated timestamp for versions to use local timezone.

### PR DESCRIPTION
**Pull request by markjeee**
Initial prototype for review (no tests yet).

### CODE REVIEW

**Comment by markjeee** [comment]
[Date / time entries as local time to user machine](https://trello.com/c/eJVvTnfX/42-date-time-entries-as-local-time-to-user-machine)

**Comment by markjeee** [comment]
[Field name used to be 'username' for the gems/<>/versions API.](https://trello.com/c/RFuz7vXl/54-field-name-used-to-be-username-for-the-gems-versions-api)

File: lib/gemfury/command/app.rb [hunk 1]
```diff
@@ -292,4 +292,19 @@ def die!(msg, err = nil, command = nil)
     shell.say %Q(#{err.class.name}: #{err}\n#{err.backtrace.join("\n")}) if err && ENV['DEBUG']
     exit(1)
   end
+
+  def time_ago(tm)
```
Comment (rykov) [inline lib/gemfury/command/app.rb]: I was thinking that we always show the timestamp, and then append a user-friendly hint if it's less than 24 hours ago.  Something like this, what do you think:

```
 def time_ago(tm)
    out = tm.strftime('%F %R')
    in_secs = Time.now - tm

    out << if in_secs < 60
      ' (%d secs ago)' % in_secs
    elsif in_secs < 3600
      ' (%s mins ago)' % (in_secs / 60).ceil
    elsif in_secs < (3600 * 24)
      ' (%s hours ago)' % (in_secs / 3600).ceil
    end

    return out
end
```

File: lib/gemfury/command/app.rb [hunk 2]
```diff
@@ -48,7 +48,7 @@ def versions(gem_name)
       va = []
       va = [ %w{ version uploaded uploaded_by } ]
```
Comment (rykov) [inline lib/gemfury/command/app.rb]: If `time_ago` returns variable length (see comment below), maybe the `uploaded_at` column should be last here.

**Pull request merged**